server/views/pages/activities/daily-attendance-summary/attendances.njk

```diff
@@ -197,7 +198,13 @@
                     {% set sessionNameHtml = attendee.attendance.timeSlot + '<br>' + (attendee.attendance.startTime) + ' to ' + (attendee.attendance.endTime)  %}
                 {% endif %}
 
-                {% set attendanceDetailsHtml = attendanceTag(attendee.attendance.status, attendee.attendance.attendanceReasonCode) + ' ' + renderPayStatus(attendee.attendance) %}
+                {% set attendanceDetailsHtml = attendanceTag(attendee.attendance.status, attendee.attendance.attendanceReasonCode) + ' ' + renderPayStatus({
+                    dataPresentCheck: attendee.attendance,
+                    status: attendee.attendance.status,
+                    issuePayment: attendee.attendance.issuePayment,
+                    payable: true,
```

Comment:
This version of the macro never has the "Unpaid" result, so if the result isn't "Pay", then it must be "No pay"